docs/src/browsers.md

```diff
@@ -157,6 +157,19 @@ See all supported browsers:
 pwsh bin\Debug\netX\playwright.ps1 install --help
 ```
 
+## Install browsers via API
+* langs: csharp
+
+It's possible to run [Command line tools](./cli.md) commands via the .NET API:
+
+```csharp
+var exitCode = Microsoft.Playwright.Program.Main(new[] {"install"});
```

Comment:
Is this an official way to run our program? Do we guarantee it's not going to change?

Reply:
yes, we do.